### ISSUE TITLE
[v4.6.0] bug: ion-searchbar style crashed when reopen the page

### ISSUE THREAD

**Issue by petrot** [ionic-team/ionic#18642]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**
ion-searchbar style crashed when reopen the page

**Expected behavior:**
works normal

**Steps to reproduce:**
- create a starter app with `ionic start searchBug sidemenu`
- add the ion-searchbar content inside an ion-toolbar tag to the homePage header

```
<ion-header>
  <ion-toolbar>
    <ion-buttons slot="start">
      <ion-menu-button></ion-menu-button>
    </ion-buttons>
    <ion-title>
      Home
    </ion-title>
  </ion-toolbar>
  <ion-toolbar no-padding class="search-toolbar">
    <ion-searchbar #searchbar
      placeholder="Search" animated="true" cancelButtonText="Cancel" [showCancelButton]="true">
    </ion-searchbar>
  </ion-toolbar>
</ion-header>
```

Now run the app with `ionic serve`

The app starts with the homepage. Now navigate to the listPAge then navigate back to the homePage (with the sidemenu). Now the searchbar crashed.

https://www.dropbox.com/s/mkiwhk7wx0ketru/Screenshot%202019-06-27%2015.13.41.png?dl=0

**Other information:**

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 5.1.1 (/Users/tomi/.nvm/versions/node/v10.14.1/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.6.0
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Utility:

   cordova-res : 0.3.0 (update available: 0.5.1)
   native-run  : 0.2.7 

System:

   NodeJS : v10.14.1 (/Users/tomi/.nvm/versions/node/v10.14.1/bin/node)
   npm    : 6.9.0
   OS     : macOS Mojave
```

**Comment by brandyscarney** [comment]
Thanks for the issue! I am going to close this as it has been fixed in Ionic, it will be in the 4.6.1 hotfix release. Please follow #18655 for updates on this. 🙂

**Comment by petrot** [comment]
@brandyscarney ok, thanks! :)
